apps/include/system/dat_init.h

```diff
@@ -0,0 +1,46 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
```

Comment:
Request to please change the year wherever possible.

Reply:
Since it's last year's commit, have kept the year intact.